br/pkg/lightning/backend/local/localhelper.go

```diff
@@ -428,9 +450,15 @@ func (local *local) waitForScatterRegion(ctx context.Context, regionInfo *split.
 	for i := 0; i < split.ScatterWaitMaxRetryTimes; i++ {
 		ok, err := local.isScatterRegionFinished(ctx, regionID)
 		if err != nil {
-			log.L().Warn("scatter region failed: do not have the region",
-				logutil.Region(regionInfo.Region))
-			return
+			log.L().Warn(
+				"wait for scatter region failed, will scatter again",
+				logutil.Region(regionInfo.Region),
+				zap.Error(err),
+			)
+			if err := local.splitCli.ScatterRegion(ctx, regionInfo); err != nil {
```

Comment:
maybe we can retry scatter only on retryable error

Reply:
Fixed in 851ada81af9165c63f2e54e28e6b60eb30e217a8.